### DISCUSSION TITLE
5.02 Configuration help request

### DISCUSSION THREAD

**Discussion by DrR0Ck** [haugene/docker-transmission-openvpn#2597] [Q&A]
Hi Folks!  Been using this for a very long time on a Synology NAS - so long that I can barely recall setting it up in the first place.  I'm basically a Linux newb, but willing to learn.  All of which is to preface that I discovered with the recent introduction of v. 5, there are no longer file/folder mapping options on a Volume tab as there were in versions through v4.x.  In my existing containers, I have folders mapped for /data, /data/completed, /data/watch, /data/incomplete, and /data/transmission-home.  And I have /etc/resolv.conf mapped and set to read only.  I gather that these can be set up using environment variables now, but may not need to be.  So I am hoping someone can walk me through setting these up. such that my existing container settings can be migrated to the new 5.0.x version.  I've tried reviewing the documentation, but unfortunately, it's not clear what I need to do.  Any help would be appreciated.  Thanks!   An apologies if this is not the appropriate place to ask.

**Comment by pkishino** [comment]
Missing /config Mount

**Reply by DrR0Ck** [reply]
Hi!  Thanks for your response.  I took a look at the documentation, but it's not clear to me specifically what I need to do.  If a tutorial exists somewhere, or if you or some other kind person could take the time to walk me through, I would be appreciative.

**Reply by DrR0Ck** [reply]
I know this has been pending a=for a long time, during which, I was beating my head against the wall trying to figure this out.  So - I think my issue may have been due to the DSM 7 upgrade I performed.  The folder/file mapping now occurs near the end of the container setup process, and hasn't been removed as I had thought.  So chalk it up to me being a bit dense.  I'm good to go and this can be closed.
